packages/Table/src/Table.styles.js

```diff
@@ -59,6 +59,8 @@ export const TD = styled.td(({ borderType }) => {
 export const TH = styled.th(({ borderType }) => {
   return css`
     ${borderType in borderTypesStyles ? borderTypesStyles[borderType] : ""}
+    font-size: 16px;
```

Comment:
we can use the font scale here

```
  ${stylers.fontSize(0)};
```

Reply:
Actually would need to use `helpers.fontScale()` here as it's a Js file. In general I'm seeing quite a few components neglecting to use this so figured we were not mandating it but I think it's a good idea as well.